src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -14,6 +28,24 @@
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private void resize() {
+        Node<K, V>[] currentTable = table;
+        table = (Node<K, V>[]) new Node[currentTable.length * RESIZE_NUMBER];
+
+    }
```

Comment:
The `resize` method implementation is incomplete. You need to re-hash the entries and move them to the new table. Also, remember to update the threshold for resizing after the table has been expanded.